server/CVCMIServer.cpp

```diff
@@ -170,10 +170,17 @@ void CVCMIServer::run()
 #endif
 
 		startAsyncAccept();
+		
+#ifndef VCMI_ANDROID
 		if(shm)
 		{
 			shm->sr->setToReadyAndNotify(port);
 		}
+#else
+		CAndroidVMHelper vmHelper;
+		vmHelper.callStaticVoidMethod(CAndroidVMHelper::NATIVE_METHODS_DEFAULT_CLASS, "onServerReady");
+		vmHelper.Detach();
```

Comment:
same here. Declare local scope

Reply:
Current scope end on the next line anyway. I added Detach to be more explicit. If I remove Detach should I add inner scope?